deployment/node-problem-detector.yaml

```diff
@@ -18,6 +18,13 @@ spec:
         - --logtostderr
         - --system-log-monitors=/config/kernel-monitor.json,/config/docker-monitor.json
         image: k8s.gcr.io/node-problem-detector:v0.5.0
+        resources:
+          limits:
+            cpu: 300m
+            memory: 100Mi
+          requests:
+            cpu: 300m
+            memory: 100Mi
```

Comment:
I tested v0.6.0 that the resource consumption is:
- CPU: 2m
- MEM: 30Mi

Maybe we should lower the numbers to something smaller, like cpu 10m and memory 50Mi to better reflect the resource consumption of NPD? WDYT?

Reply:
According to your test, it should be ok to run NPD with cpu 10m. I suggest we add more memory to avoid OOM. @wangzhen127

Reply:
Finanlly, cpu requests/limits is 10m, memory requests/limits is 80Mi.